numpy/ma/tests/test_extras.py

```diff
@@ -292,7 +292,20 @@ def test_complex(self):
         assert_almost_equal(wav1.real, expected1.real)
         assert_almost_equal(wav1.imag, expected1.imag)
 
-
+    def test_masked_weights(self):
+            # Test with masked weights.
+            # (Regression test for https://github.com/numpy/numpy/issues/10438)
+            a = np.ma.array(np.arange(9).reshape(3, 3), mask=[[1, 0, 0], [1, 0, 0], [0, 0, 0]])
+            weights_unmasked = masked_array([5,28,31],mask=False)
```

Comment:
This will make the linter a little unhappy, you can see all the PEP8 issues that will show up in CI with `python tools/linter.py` (run in the root of the repo).

Reply:
@rgommers, the linting issues should now be fixed

Reply:
The linter is indeed happy. Now on to the next step:)